source/common/upstream/cds_api_helper.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "envoy/config/subscription.h"
+#include "envoy/upstream/cluster_manager.h"
+
+#include "common/common/logger.h"
+#include "common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Upstream {
+
+class CdsApiHelper : Logger::Loggable<Logger::Id::upstream> {
```

Comment:
Please add some class comments on what this class is for to aid review/readability.

Reply:
Addressed in https://github.com/envoyproxy/envoy/pull/15806.